sdk/src/main/java/com/bugsnag/android/JsonStream.java

```diff
@@ -69,7 +69,7 @@ public void value(@NonNull File file) throws IOException {
         }
 
         super.flush();
-        beforeValue(false); // add comma if in array
+        beforeValue(); // add comma if in array
```

Comment:
why did this need to change?

Reply:
The boolean parameter for this method has been [removed](https://github.com/bugsnag/bugsnag-android/pull/415/files#diff-f33a35f88c8e8350f363fd92e8e25e0cL679) since we last updated. Passing in a parameter which hasn't been explicitly specified would lead to a compilation error in Java.